ngraph/core/src/op/deformable_convolution.cpp

```diff
@@ -53,139 +53,174 @@ bool op::v1::DeformableConvolution::visit_attributes(AttributeVisitor& visitor)
 void op::v1::DeformableConvolution::validate_and_infer_types()
 {
     NGRAPH_OP_SCOPE(v1_DeformableConvolution_validate_and_infer_types);
-    const PartialShape& data_batch_shape = get_input_partial_shape(0);
-    const PartialShape& deformable_values_shape = get_input_partial_shape(1);
-    const PartialShape& filters_shape = get_input_partial_shape(2);
+    const PartialShape& data_batch_pshape = get_input_partial_shape(0);
+    const PartialShape& deformable_values_pshape = get_input_partial_shape(1);
```

Comment:
again here, `deformable_values_pshape` can be confusing as it is not present in spec, better to name it `offsets_pshape`

Reply:
Done